mountpoint-s3/CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
-## Unreleased
+## v1.8.0
 
 ### New features
 
 * Mountpoint now offers two new command-line arguments `--read-part-size <SIZE>` and `--write-part-size <SIZE>` which allow to specify different part sizes to be used when reading and writing respectively.
```

Comment:
We forgot to add the link to the PR here
```suggestion
* Mountpoint now offers two new command-line arguments `--read-part-size <SIZE>` and `--write-part-size <SIZE>` which allow to specify different part sizes to be used when reading and writing respectively. ([#949](https://github.com/awslabs/mountpoint-s3/pull/949))
```

Reply:
My bad :)

Reply:
Not really, I reviewed that PR and just realised we should add it :(

Reply:
done